packages/mobile/src/config/routing/TabNavigation.tsx

```diff
@@ -40,7 +40,7 @@ const kvArray = [
           </AlertView>
       </AlertV>
   ],
-  [NavigationName.FILTERSSCREEN, <SearchWIc source={{uri:'ic_searchW'}}/>],
+  [NavigationName.FILTERSSCREEN, <SearchWIc source={{uri:'ic_searchw'}}/>],
```

Comment:
Ten AlertV wydzielic do osobnego komponentu

Reply:
tabBarIcon zwraca any i ta konstrukcja z Map jest dziwna

Reply:
czemu dziwna ? robie tam mapa, który z routa bierze nazwe i po nazwie dopasowuje ikonke do dolnego menu.  Dlatego jest to tak zrobione, żeby Tabs.Navigator nie miał wielkiej tablicy z opcjami tylko to
  screenOptions={this.screenOptions}